src/Knp/FriendlyContexts/Utils/TextFormater.php

```diff
@@ -54,6 +54,11 @@ public function tableToString(array $array)
 
     public function listToArray($list, $delimiters = [', ', ' and '], $parser = "#||#")
     {
+        $arrayFromJson = json_decode($list, true);
```

Comment:
You should first check that `$list` isn't numeric because `json_decode` won't raise any issue in that case.
![image](https://cloud.githubusercontent.com/assets/254782/26683328/b61392ea-46e3-11e7-92b9-ce15d3ef7749.png)